instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
 import './CommentSection.css';
+import moment from 'moment';
 
 
 class CommentSection extends React.Component {
+   
+    constructor(props) {
+        super(props)
+        this.addNewComment = this.addNewComment.bind(this);
```

Comment:
Note that instead of binding the this keyword in the constructor, you can instead turn addNewComment into an arrow function